build/stf-run-ci/tasks/main.yml

```diff
@@ -22,18 +32,44 @@
   loop:
     - smartgateways.smartgateway.infra.watch
     - servicetelemetrys.infra.watch
+    - lokistacks.loki.openshift.io
 
 - block:
   - name: Setup supporting repositories
     include_tasks: clone_repos.yml
     tags:
       - clone
 
+  - name: Get new operator sdk
+    command: ./get_new_operator_sdk.sh
```

Comment:
If you make the version configurable in the script, I'd pass an argument here with the version you expect.